packages/plugin-ext/src/main/node/plugin-ext-backend-module.ts

```diff
@@ -25,6 +25,7 @@ import {
 } from '../../common/plugin-protocol';
 import { PluginDeployerImpl } from './plugin-deployer-impl';
 import { LocalDirectoryPluginDeployerResolver } from './resolvers/plugin-local-dir-resolver';
```

Comment:
The file was deleted. There is nowhere for this to import from.

I believe there's more instant of this file being imported. If you delete the file, please resolve these instants.